tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "checkJs": true,
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "noImplicitAny": false,
+    "outDir": "lib",
+    "skipLibCheck": true,
```

```suggestion
    "skipLibCheck": false,
```

Unless there's a good reason to have it.

---

I don't know if it is a good reason but without I have this error:
```
node_modules/vite/dist/node/index.d.ts:5:41 - error TS2307: Cannot find module 'rollup/parseAst' or its corresponding type declarations.
  There are types at '/Users/lpatiny/git/cheminfo/zenodo/node_modules/rollup/dist/parseAst.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.

5 export { parseAst, parseAstAsync } from 'rollup/parseAst';
```

---

Yeah, that's an annoying issue, but I don't see where you import vite in this project.

---

The solution is to migrate to ESM-first or ESM-only but that's not a simple task, especially if you have never done it before.